### PR TITLE
feat: add support for bumping version # in bower.json

### PR DESCRIPTION
**Pull request by bcoe**
working from @lexich's foundation, this pull adds support for bumping version # in multiple config files in a generic way.

### CODE REVIEW

**Comment by bcoe** [comment]
@lexich I've released support for `bower.json` in the upcoming release of `standard-version`; I would love the extra set of eyes/help testing:

`npm cache clear; npm i standard-version@next`

**Comment by lexich** [comment]
@bcoe nice. I think that the next step should be to adding support pluggins for different file formats.

File: index.js [hunk 1]
```diff
@@ -52,6 +47,37 @@ module.exports = function standardVersion (argv, done) {
...
+        var filename = path.basename(configPath)
+        config.version = newVersion
+        fs.writeFileSync(configPath, JSON.stringify(config, null, 2) + '\n', 'utf-8')
+        checkpoint(args, 'bumping version in ' + filename + ' from %s to %s', [config.version, newVersion])
+        // flag any config files that we modify the version # for
+        // as having been updated.
+        configsToUpdate[configPath] = true
+      }
+    } catch (err) {
+      if (err.code !== 'ENOENT') console.warn(err.message)
```
Comment (Tapppi) [inline index.js]: @bcoe This PR is so cool, thanks for getting back into the project so strongly, I've been AWOL due to increased pressure at our startup but am trying to catch up too. 

I'm slightly concerned about the fact that any write errors will be ignored, e.g. if you have a `bower.json` you would presumably want to have it updated too. If the write fails, you'll just get a error but the git commits etc. will go through!

I haven't yet settled on an opinion on how this should be handled, but I think it's part of a bigger problem related to rolling back changes if an error occurs during the whole run..

Comment (e-cloud) [inline index.js]: rolling back changes will be a awesome feature

**Pull request merged**